rmgpy/kinetics/surface.pyx

```diff
@@ -78,6 +84,11 @@ cdef class StickingCoefficient(KineticsModel):
         string = 'StickingCoefficient(A={0!r}, n={1!r}, Ea={2!r}, T0={3!r}'.format(self.A, self.n, self.Ea, self.T0)
         if self.Tmin is not None: string += ', Tmin={0!r}'.format(self.Tmin)
         if self.Tmax is not None: string += ', Tmax={0!r}'.format(self.Tmax)
+        if self.coverage_dependence is not None:
```

Comment:
I think this shouldn't be included if it's an empty dict (at the moment we get a lot of `coverage_dependence={}` which could be skipped)

Reply:
Same for the other `__repr__` methods (this isn't the only one).

Reply:
I could change it to, 
``` if self.coverage_dependence is not {}```?

Reply:
Just "if self.coverage_dependence:" ?
None and an empty dict both evaluate to Boolean False